packages/mllp-server/src/app.ts

```diff
@@ -47,19 +48,25 @@ async function createHl7Server(logger: Logger): Promise<Hl7Server> {
         );
 
         const { cxId, patientId } = getCxIdAndPatientIdOrFail(message);
+        const { messageCode, triggerEvent } = getHl7MessageTypeOrFail(message);
 
-        const msgType = getHl7MessageTypeOrFail(message);
         Sentry.setExtras({
           cxId,
           patientId,
-          messageType: msgType.messageType,
-          messageCode: msgType.triggerEvent,
+          messageCode,
+          triggerEvent,
         });
```

Comment:
Since we're touching this, can we change these calls to use Core's `capture.setExtra`, please?

Reply:
yep